### DISCUSSION TITLE
Move encryption algorithm `ALGORITHM = "HS256"` to config

### DISCUSSION THREAD

**Discussion by sameeramin** [fastapi/full-stack-fastapi-template#1159] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
N/A
```

### Description

I've noticed that we have been defining the encryption algorithm variable more than once and there's no single source of truth. I propose to move it to config to solve this problem.

### Operating System

macOS

### Operating System Details

MacOS Sonomo

### Python Version

3.10

**Comment by YuriiMotov** [comment, marked as answer]
Currently `ALGORITHM = "HS256"` is defined only once in `backend/app/core.security.py`
